unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -89,24 +89,12 @@ public void Update()
             cameraForward = CommonScriptableObjects.cameraForward.Get();
 
             UpdateAllLODs(maxAvatars.Get(), maxImpostors.Get());
-            UpdateLODsBillboard();
+            UpdateLODsBillboardOrientation();
         }
 
-        internal void UpdateLODsBillboard()
+        internal void UpdateLODsBillboardOrientation()
         {
-            foreach (var kvp in lodControllers)
-            {
-                Player player = kvp.Value.player;
-
-                if (!IsInFrontOfCamera(player.worldPosition))
-                    continue;
-
-                Vector3 previousForward = player.forwardDirection;
-                Vector3 lookAtDir = (cameraPosition - player.worldPosition).normalized;
-
-                lookAtDir.y = previousForward.y;
-                player.renderer.SetImpostorForward(lookAtDir);
-            }
+            //Debug.Log("TODO"); // Move to a component or the LOD itself
```

Comment:
let's remove this comment

Reply:
Oh! I forgot to implement the impostors orientation!

I will leave that comment until is done and then remove it!

Reply:
Take a look again @pravusjif fixed this!